corehq/apps/app_manager/tests/data/suite/autoload_supplypoint.xml

```diff
@@ -14,14 +14,7 @@
              function="instance('commcaresession')/session/user/data/commtrack-supply-point"/>
     </session>
     <assertions>
-      <assert test="count(instance('commcaresession')/session/user/data/commtrack-supply-point) = 1">
-        <text>
-          <locale id="case_autoload.location.property_missing">
-            <argument>commtrack-supply-point</argument>
-          </locale>
-        </text>
-      </assert>
-      <assert test="count(instance('casedb')/casedb/case[@case_id=instance('commcaresession')/session/user/data/commtrack-supply-point]) = 1">
+      <assert test="if(count(instance('commcaresession')/session/user/data/commtrack-supply-point) = 1, count(instance('casedb')/casedb/case[@case_id=instance('commcaresession')/session/user/data/commtrack-supply-point]) = 1, true())">
```

Comment:
Will this still not raise an error loading the datum if there is no `commtrack-supply-point` element?

I think the function expression for the datum may also have to be an if.

Also, if the user has no user data at all then the `instance('commcaresession')/session/user/data` won't exist either which is what caused the original bug.